guides/release/in-depth-topics/autotracking-in-depth.md

```diff
@@ -321,4 +321,84 @@ class ShoppingList {
 }
 ```
 
+## Caching of tracked properties
+
+In contrast with computed properties from pre-Octane, autotracked properties
+are not cached. This means that an autotracked property will always be
+recomputed, regardless of whether its dependencies have changed. The following
+example shows how this works:
+
+```js
+import { tracked } from '@glimmer/tracking';
+
+let count = 0;
+
+class Person {
+  @tracked firstName = 'Jen';
+  @tracked lastName = 'Weber';
+
+  get fullName() {
+    ++count;
+    return `${this.firstName} ${this.lastName}`;
+  }
+}
+
+let person = new Person();
+
+console.log(person.fullName); // Jen Weber
+console.log(count); // 1;
+console.log(person.fullName); // Jen Weber
+console.log(count); // 2;
+
+person.firstName = 'Jennifer';
+
+console.log(person.fullName); // Jennifer Weber
+console.log(count); // 3;
+```
+
+This works fine in most cases. But sometimes you want a bit more control over
+the recomputation, for instance if the computation that happens in the getter
+is very expensive. In that case, you want to cache the value in between calls
+when the dependencies haven't changed. You want to recompute only if one of the
+dependencies has been modified.
+
+The way to achieve this is through Ember's [cache
+API](https://github.com/emberjs/rfcs/blob/master/text/0615-autotracking-memoization.md).
+Using `createCache`, we can cache an autotracked property. The property can
+then be consumed with the `getValue` function. The way it works is as follows:
```

Comment:
I added a link to the polyfill.

According to the [RFC](https://github.com/emberjs/rfcs/blob/master/text/0615-autotracking-memoization.md#usage), I think `createCache` receives a function. I updated your Guides text to make this clear:

```suggestion
Ember's [cache API](https://github.com/ember-polyfills/ember-cache-primitive-polyfill) lets you cache a getter in 2 steps.
First, you pass a function that is costly to compute to `createCache`.
Then, in the getter, you call the function with `getValue` and return its value.

With these 2 steps in mind, let's introduce caching to `aspectRatio`:
```

Reply:
Nice!